### ISSUE TITLE
Dropzone "browse files" should be a button instead of a link

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#1162]
Currently the only KB accessible control within a Dropzone is the "browse files" link:

![image](https://user-images.githubusercontent.com/756393/76980237-dcaea080-6930-11ea-9957-652d5b907a66.png)

```
<a class="dropzone__browse" href="#">Browse Files…</a>
```

This should be changed to a button as the existing link triggers a file select dialog and doesn't navigate anywhere. We should also look to improve the default text so it has more meaning when read as part of a form.
